test/e2e/specs/field-render-tests.js

```diff
@@ -157,7 +160,7 @@ module.exports = {
     browser.click('#form-demo') // click outside calender (work around for IE11 issue: https://github.com/chmln/flatpickr/issues/900)
     browser.expect.element('.flatpickr-calendar').to.be.not.visible
 
-    const today = new Date().toJSON().slice(0, 10)
+    const today = moment().format('YYYY-MM-DD');
```

Comment:
There's two other variables called today in this file that get instantiated in different ways.
This should probably the fix for all three.